src/app/controllers/user/user.controller.ts

```diff
@@ -1,10 +1,62 @@
 module webShoes {
+  'use strict';
```

Comment:
@adrianovalente They will, but the idea is to keep it so we write beautiful code. I'll delete some of the old folders which i didn't care about coding style and make sure the new ones follow the style. Most of it it's just typing and replacing "" for ''